src/metabase/query_processor/util/add_alias_info.clj

```diff
@@ -186,19 +191,34 @@
             join))
         joins))
 
+(defn- matching-field-in-source-query* [source-query field-clause & {:keys [normalize-fn]
+                                                                     :or   {normalize-fn normalize-clause}}]
+  (let [normalized (normalize-fn field-clause)
+        exports    (filter (partial mbql.u/is-clause? :field)
+                           (exports source-query))]
+    ;; first look for an EXACT match in the `exports`
+    (or (some (fn [a-clause]
+                (when (= (normalize-fn a-clause) normalized)
+                  a-clause))
+              exports)
+        ;; if there is no EXACT match, attempt a 'fuzzy' match by disregarding the `:temporal-unit` and `:binning`
+        (let [fuzzify          (fn [clause] (mbql.u/update-field-options clause dissoc :temporal-unit :binning))
```

Comment:
nit: function body to next line/indented?

Reply:
Why? https://guide.clojure.style/#one-line-functions